train/runExe.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace RunExe
+{
+    class Program
+    {
+        static void Main()
+        {
+            var proc = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = "c://users/admin/source/repos/DirectoryStatistics/DirectoryStatistics/bin/Debug/netcoreapp3.1/DirectoryStatistics.exe",
+                    Arguments = "-d /Users/admin/Desktop/dailyreport -p *.txt",
+                    UseShellExecute = false,
+                    RedirectStandardOutput = true,
+                    CreateNoWindow = true
```

Comment:
I think you need RedirectStandardError here to capture the potential error message